debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/SourceInfoTest.java

```diff
@@ -165,15 +165,32 @@ public void shouldRecoverSourceInfoFromOffsetWithFilterData() {
 
         sourceWith(offset);
         assertThat(source.hasFilterInfo()).isTrue();
-        assertEquals(databaseWhitelist, source.getDatabaseWhitelist());
-        assertEquals(tableWhitelist, source.getTableWhitelist());
+        assertEquals(databaseWhitelist, source.getDatabaseIncludeList());
+        assertEquals(tableWhitelist, source.getTableIncludeList());
         // confirm other filter info is null
-        assertThat(source.getDatabaseBlacklist()).isNull();
-        assertThat(source.getTableBlacklist()).isNull();
+        assertThat(source.getDatabaseExcludeList()).isNull();
+        assertThat(source.getTableExcludeList()).isNull();
     }
 
     @Test
-    public void setOffsetFilterFromFilter() {
+    public void shouldRecoverSourceInfoFromOffsetWithFilterData() {
+        final String databaseWhitelist = "a,b";
```

Comment:
Should use new names? The only time a var should use the old names is when it actually refers to the old property/key in some backwards compat test.

Reply:
same here as well!

Reply:
in https://issues.redhat.com/browse/DBZ-2433 I described it like old property tests can have old names, but we always prefer new / non-divisive names.